scripts/test.py

```diff
@@ -42,21 +42,20 @@ def monte_carlo_sampling_chaospy(N_FEATURES, SAMPLES, DISTRIBUTION, rule="latin_
     import chaospy
     from scipy.stats import qmc 
     from sklearn.preprocessing import MinMaxScaler
-
-
+    params = tuple(DISTRIBUTION_PARAMS)
     # Generate a Nfeatures-dimensional latin hypercube varying between 0 and 1:
-    N_FEATURES = f"chaospy.{DISTRIBUTION}(0, 1), "*N_FEATURES
+    N_FEATURES = f"chaospy.{DISTRIBUTION}{tuple(params)}, "*N_FEATURES
```

Comment:
Is `tulpe()` needed be repeated here? It looks like `param` is defined as `tulpe` just above by the line 45.

Reply:
I didnt catch this 👍🏼